runHarness/DataManagers/AuctionKubernetesDataManager.pm

```diff
@@ -141,6 +141,11 @@ sub stopDataManagerContainer {
 	$cluster->kubernetesDelete("configMap", "auctiondatamanager-config", $self->appInstance->namespace);
 	$cluster->kubernetesDelete("deployment", "auctiondatamanager", $self->appInstance->namespace);
 
+	# Don't return until the data manager pod has terminated
+    my $podExists = 1;
+    do {
+	  $podExists = $self->host->kubernetesDoPodsExist("impl=auctiondatamanager", $self->appInstance->namespace );    	
+    } while ($podExists);
 }
```

Comment:
Should this have a small sleep in the loop and a max retries?

Reply:
Yes and yes.  I will add these

Reply:
I added a sleep and max retries.  Even if the pod hasn't terminated after the retries I let the run proceed.  usually this won't be an issue.